style.css

```diff
@@ -0,0 +1,54 @@
+#something {
+  text-align: center;
+}
+
+#somethinmore {
+  align-items: center;
+  padding-bottom: 2em;
+  padding-top: 2em;
+}
+
```

Comment:
Please use Flexbox to align this section as is required to approve this project and makes it look nicer.
Your end result should look like the one in the template:

![shipping](https://user-images.githubusercontent.com/111214189/192381957-5fec831a-dac4-4a9b-8c24-937b57a62991.png)